ibm/mas_devops/roles/suite_dns/README.md

```diff
@@ -121,6 +121,26 @@ Role Variables - IBM Cloud Internet Services DNS Integration
 - Environment Variable: `CIS_SUBDOMAIN`
 - Default: None
 
+### cis_enhanced_security
+Set this to true to enable the enhanced CIS DNS integration security. See below for details.
+
+- Optional
+- Environment Variable: `CIS_ENHANCED_SECURITY`
+- Default: false
+
+
+Role Variables - Enhanced IBM CIS DNS Integration Secruity
+------------------------------------------------------------
+- Enable WAF firewall disabling rules that affect MAS application functionalities
+- Enable Proxy for DNS entries
+- Ensure there are no wildcard DNS entry in CIS
+- Create Edge Certificates in CIS instance
+
+### ibmcloud_api_key
```

Comment:
We already have `ibmcloud_apikey` .. which is used when provisioning clusters ... what is the purpose of this variable?

If this is passed into an automated process that communicates with CIS can we use the existing `cis_apikey`?  If this is used as one-time (when the role runs) authentication to IBM Cloud, can we use ibmcloud_apikey, so that we don't have ibmcloud_apikey and ibmcloud_api_key.